lib/linux.js

```diff
@@ -75,6 +75,14 @@ const appsList = [
 			'/org/mate/desktop/background/picture-filename'
 		],
 		transform: imagePath => imagePath.slice(1, -1)
+	},
+	{
+		cmd: 'qdbus',
+		set: ['org.kde.plasmashell',
+			'/PlasmaShell',
+			'org.kde.PlasmaShell.evaluateScript',
+			'var allDesktops = desktops();print (allDesktops);for (i=0;i<allDesktops.length;i++) {d = allDesktops[i];d.wallpaperPlugin = "org.kde.image";d.currentConfigGroup = Array("Wallpaper", "org.kde.image", "General");d.writeConfig("Image", "file://%s")}'
```

Comment:
Use a template literal here for the code and format it properly, for readability reasons.

Reply:
What JS engine and version is `PlasmaShell` running?

Reply:
Not sure what you mean with template literal here (haven't worked much with JS).
Also don't know much about the JS engine.
Just wanted to share my solution here after I managed to find/create a working command.

Reply:
https://www.google.co.th/search?q=javascript+template+literal&oq=javascript+template+literal&aqs=chrome..69i57j0l5.7768j0j1&sourceid=chrome&ie=UTF-8

Reply:
I found them of course :D Just not sure how to apply them here.
Just use the multiline (without \n) functionality like follows or do some magic tricks with replacement and more?

```
set: ['org.kde.plasmashell',
		'/PlasmaShell',
		'org.kde.PlasmaShell.evaluateScript',
		`var allDesktops = desktops();
		//print (allDesktops);
		for (i=0;i<allDesktops.length;i++) {
		d = allDesktops[i];
		d.wallpaperPlugin = "org.kde.image";
		d.currentConfigGroup = Array("Wallpaper", "org.kde.image", "General");
		d.writeConfig("Image", "file://%s");
		}` ]
```

Reply:
That's what I was thinking, yes.